docs/whats_new.rst

```diff
@@ -51,6 +51,22 @@ Django Channels
 
 See: :doc:`./tutorials/getting_started/websockets`
 
+Bokeh Integration
+-----------------
+
+* Bokeh's functionality to link backend Python objects to frontend JavaScript objects has been added.
+
+See: `Bokeh Integration Documentation <./tethys_sdk/url_maps.html#bokeh-integration>`_
+
+See: `Bokeh Integration Tutorial <./tutorials/getting_started/bokeh_integration.html>`_
```

Comment:
Do these links work? I'd prefer if you used the ref/doc link format, which is demonstrated many times in this document.

Reply:
If you look closely, the first link is referencing a sub section. I don't think you can do that with ref or doc, it'd have to be the whole document. That's why I did it like that, but I did test it.

I did change the second link to use :doc: and added a bokeh function that was missing from the mocked modules in the documentation.

Reply:
You can do a subsections with a ref, that's what they are for. You add the ref tag right before the subsection header and then use :ref: to reference it. There are many examples all throughout the documentation.

Reply:
Ahh, got it. To use ref from another file a tag needs to be created for the subsection. Done